src/main/resources/assets/arcanuscontinuum/lang/fr_fr.json

```
@@ -0,0 +1,334 @@
+	"spell_component.arcanuscontinuum.shrink_effect": "Effet de Rétrécissement",
+	"spell_component.arcanuscontinuum.enlarge_effect": "Effet d'Elargissement",
+	"spell_component.arcanuscontinuum.copper_curse_effect": "Effet de Malédiction du Cuivre",
+	"spell_component.arcanuscontinuum.temporal_dilation_effect": "Effet de Distorsion Temporel",
+	"spell_component.arcanuscontinuum.spatial_rift_effect": "Effet de Faille Spatiale",
+	"spell_component.arcanuscontinuum.warding_effect": "Effet de Protéction",
+	"spell_component.arcanuscontinuum.discombobulate_effect": "Effet de Confusion",
+	"spell_component.arcanuscontinuum.haste_effect": "Effet de Hâte",
+	"spell_component.arcanuscontinuum.float_effect": "Effet de Flottement",
+	"spell_component.arcanuscontinuum.mana_wings_effect": "Effet d'Ailes de Mana",
+	"spell_component.arcanuscontinuum.stockpile_effect": "Effet de Renvoie d'Attaque",
+	"spell_component.arcanuscontinuum.danger_sense_effect": "Effet de Sens du Danger",
+
+  "item.arcanuscontinuum.compendium_arcanus.landing_text": "Arcanus: Continuum introduit un nouveau système de magie dans le monde de Minecraft. Les joueurs peuvent créer leurs propres sorts comme bon leur semble !",
+
+	"spell_book.arcanuscontinuum.weight": "Poids",
+	"spell_book.arcanuscontinuum.weight.none": "Auncun",
+	"spell_book.arcanuscontinuum.weight.very_light": "Très Léger",
+	"spell_book.arcanuscontinuum.weight.light": "Léger",
+	"spell_book.arcanuscontinuum.weight.medium": "Moyen",
+	"spell_book.arcanuscontinuum.weight.heavy": "Lourd",
+	"spell_book.arcanuscontinuum.weight.very_heavy": "Très Lourd",
+	"spell_book.arcanuscontinuum.mana_cost": "Coût de Mana",
+	"spell_book.arcanuscontinuum.mana_multiplier": "Mana Multiplier",
+	"spell_book.arcanuscontinuum.potency_modifier": "Potency Modifier",
+	"spell_book.arcanuscontinuum.cool_down": "Refroidissement",
+	"spell_book.arcanuscontinuum.seconds": "s",
+
+	"staff.arcanuscontinuum.invalid_data": "!!! DONNEES INVALIDES !!!",
+	"staff.arcanuscontinuum.primary_color": "Couleur des cristeaux",
+	"staff.arcanuscontinuum.secondary_color": "Couleur du bâton",
+
+	"spell.arcanuscontinuum.too_many_components": "Trop de composants!",
+	"spell.arcanuscontinuum.too_low_level": "Niveau de sorcellerie trop bas!",
+	"spell.arcanuscontinuum.not_enough_mana": "Pas assez de Mana!",
+
+	"door.arcanuscontinuum.access_granted": "Accès accordé...",
+	"door.arcanuscontinuum.not_owner": "Cette porte ne vous appartiens pas",
+	"door.arcanuscontinuum.say_magic_word": "Cette porte a besoin d'un mot de passe",
+	"door.arcanuscontinuum.password_set": "Le mot de passe defini est %s",
+
+	"screen.arcanuscontinuum.tooltip.undo": "Annuler",
+	"screen.arcanuscontinuum.tooltip.redo": "Refaire",
+	"screen.arcanuscontinuum.tooltip.component_count": "Spell Components",
+	"screen.arcanuscontinuum.tooltip.change_screens": "Changer de Mode",
+	"screen.arcanuscontinuum.tooltip.cycle_up": "Skin précedent",
```

```suggestion
	"screen.arcanuscontinuum.tooltip.cycle_up": "Skin précédente",
```
Mainly wanted to fix the accent, made it grammatically feminine too as it's technically the translation of "peau"